src/clusterfuzz/_internal/bot/tasks/setup.py

```diff
@@ -524,34 +530,38 @@ def _set_fuzzer_env_vars(fuzzer):
 def preprocess_update_fuzzer_and_data_bundles(fuzzer_name):
   """Does preprocessing for calls to update_fuzzer_and_data_bundles in
   uworker_main. Returns a SetupInput object."""
-  update_input = uworker_io.SetupInput(fuzzer_name=fuzzer_name)
-  update_input.fuzzer = data_types.Fuzzer.query(
-      data_types.Fuzzer.name == fuzzer_name).get()
-  if not update_input.fuzzer:
+  update_input = uworker_msg_pb2.SetupInput(fuzzer_name=fuzzer_name)
+  fuzzer = data_types.Fuzzer.query(data_types.Fuzzer.name == fuzzer_name).get()
+  if not fuzzer:
     logs.log_error('No fuzzer exists with name %s.' % fuzzer_name)
     raise errors.InvalidFuzzerError
 
-  update_input.data_bundles = list(
+  data_bundles = list(
       ndb_utils.get_all_from_query(
-          data_types.DataBundle.query(data_types.DataBundle.name ==
-                                      update_input.fuzzer.data_bundle_name)))
-  logs.log('Data bundles: %s' % update_input.data_bundles)
+          data_types.DataBundle.query(
+              data_types.DataBundle.name == fuzzer.data_bundle_name)))
+  logs.log('Data bundles: %s' % data_bundles)
+
+  update_input.data_bundles.extend(  # pylint: disable=no-member
```

Comment:
How come the linter complains about a missing member here? This PR seems to add a lot of such statements in places where I would not expect them.